packages/layer-tests/layer-tests.toml

```diff
@@ -3,7 +3,7 @@ log_level = ["info", "wavs=debug"]
 
 [layer-tests]
 wavs_concurrency = true
-middleware_concurrency = false
+middleware_concurrency = true
```

Comment:
🎉

Reply:
It was that easy.
I still take the credit though.

Reply:
well except that it used to break unexpectedly due to file corruption and nonce errors which you've fixed here :P